hazelcast-sql/src/main/java/com/hazelcast/jet/sql/impl/CalciteSqlOptimizer.java

```diff
@@ -606,7 +605,7 @@ private SqlPlanImpl toPlan(
             boolean isCreateJob,
             String query,
             boolean analyze,
-            Map<String, String> analyzeOptions
+            JobConfig analyzeJobConfig
```

Comment:
`analyze` can be inferred from the nullity of `analyzeJobConfig`.
```suggestion
            @Nullable JobConfig analyzeJobConfig
```

Reply:
> `analyze` can be inferred from the nullity of `analyzeJobConfig`.

What do you think @k-jamroz?

Reply:
yes, but we can keep this as is, this does not hurt that much. and OTOH flag is explicit, nullability is less obvious.